near-sdk-abi/src/lib.rs

```diff
@@ -98,6 +98,7 @@ impl Generator {
                 mod_name.map(|n| format_ident!("{}", n)),
             );
             let token_stream = quote! {
+                #![allow(unused_imports)]
                 use serde::{Deserialize, Serialize};
                 #token_stream
```

Comment:
Are these imports used? Can't seem to find where.

Reply:
Yeah unfortunately schemafy generates code that uses unqualified serde derivation. Here for example: https://github.com/Marwes/schemafy/blob/master/schemafy_lib/src/lib.rs#L560.

Reply:
Ah, makes sense.